.github/workflows/build.yml

```diff
@@ -340,14 +340,13 @@ jobs:
     steps:
       - name: Clone
         id: checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
 
       - uses: Jimver/cuda-toolkit@v0.2.10
         id: cuda-toolkit
         with:
           cuda: ${{ matrix.cuda }}
-          # TODO(green-sky): _dev seems to fail, and non dev are not enought
-          #sub-packages: '["nvcc", "cudart", "cublas", "cudart_dev", "cublas_dev"]'
+          sub-packages: '["nvcc", "cudart", "cublas", "cublas_dev", "thrust", "visual_studio_integration"]'
```

Comment:
did you get this to work? also, can you create a manual release for this pr in your repo?

Reply:
The build passed successfully.
Just create a release:
https://github.com/alonfaraj/llama.cpp/releases
I just fixed the folder name and check it when it done

Reply:
CUDA 12.2 + 11.7
https://github.com/alonfaraj/llama.cpp/releases/tag/freebsd-ci-b1-03a5465

CUDA 12.1 + 11.7
https://github.com/alonfaraj/llama.cpp/releases/tag/freebsd-ci-b1-8f26cb0

Reply:
oh god, the cuda runtime for windows grew another 40mb